src/Storages/MergeTree/ReplicatedMergeTreeSink.cpp

```diff
@@ -20,6 +21,11 @@ namespace ProfileEvents
 namespace DB
 {
 
+namespace FailPoints
+{
+    extern const char rmt_commit_zk_fail_after_op[];
```

Comment:
`rmt` seems not ClickHouse-like identifier. `replicated_merge_tree_commit_zk_fail_after_op` :-(